<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogInWidget</class>
 <widget class="QWidget" name="LogInWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>412</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>537</width>
    <height>412</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>537</width>
    <height>412</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LogInWidget</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/LogInWidget.qrc">
    <normaloff>:/LogInWidget/image/icon.png</normaloff>:/LogInWidget/image/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="backgroundWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/LogInWidget/image/login_background.jpg);</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>372</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>43</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>43</height>
       </size>
      </property>
      <property name="text">
       <string>账号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="userNameLineEdit">
      <property name="minimumSize">
       <size>
        <width>313</width>
        <height>43</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>313</width>
        <height>43</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>372</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>43</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>43</height>
       </size>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordLineEdit">
      <property name="minimumSize">
       <size>
        <width>313</width>
        <height>43</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>313</width>
        <height>43</height>
       </size>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="logInButton">
   <property name="geometry">
    <rect>
     <x>202</x>
     <y>320</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);
font: 16pt &quot;Microsoft YaHei&quot;;
color: #ffffff;</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Microsoft YaHei&quot;;
color: #808080;</string>
   </property>
   <property name="text">
    <string>注册账号</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res/LogInWidget.qrc"/>
 </resources>
 <connections/>
</ui>
